backend/src/main/java/org/hackbrooklyn/plaza/service/impl/AnnouncementServiceImpl.java

```diff
@@ -51,4 +52,14 @@ public int createNewAnnouncement(String body, User author) {
         return newAnnouncement.getId();
 
     }
+
+    @Override
+    public int updateAnnouncement(int id, String body) {
+        Announcement announcement = announcementRepository.getOne(id);
+        announcement.setBody(body);
+        announcement.setLastUpdated(LocalDateTime.now());
+        announcementRepository.save(announcement);
+
+        return announcement.getId();
```

Comment:
Since we're not returning 201 Created, we don't need to retrieve the announcement's ID and can safely return `void` wherever needed.